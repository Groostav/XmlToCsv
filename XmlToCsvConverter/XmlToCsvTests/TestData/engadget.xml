<?xml version="1.0" encoding="utf-8"?>
<rss xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:itunes="http://www.itunes.com/dtds/podcast-1.0.dtd" version="2.0">
  <channel>
    <title>Engadget RSS Feed</title>
    <link>http://www.engadget.com</link>
    <description>Engadget</description>
    <image>
      <url>http://www.blogsmithmedia.com/www.engadget.com/media/feedlogo.gif</url>
      <title>Engadget</title>
      <link>http://www.engadget.com</link>
    </image>
    <language>en-us</language>
    <copyright>Copyright 2014 AOL Inc. The contents of this feed are available for non-commercial use only.</copyright>
    <generator>Blogsmith http://www.blogsmith.com/</generator>
    <item>
      <title><![CDATA[Samsung's 'Unpacked 5' event promises a new flagship device]]></title>
      <link>http://www.engadget.com/2014/02/03/samsung-unpacked-5-invitation/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://o.aolcdn.com/hss/storage/adam/4367fa0d055ab4cd6a4a1d3be6000c2a/samsung-unpacked-5-invitation_thumbnail.jpg"/> Samsung sure knows how to whet the appetite of its fans. The Korean tech giant has announced that it's holding an "Unpacked 5" event at Mobile World Congress on February 24th -- a potentially not-so-subtle reference to the upcoming Galaxy S 5. ...]]></description>
      <dc:creator><![CDATA[Jon Fingas]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 21:52:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/samsung-unpacked-5-invitation/</guid>
    </item>
    <item>
      <title><![CDATA[Google's so-called mystery barge must relocate in light of permit blunder (updated)]]></title>
      <link>http://www.engadget.com/2014/02/03/google-mystery-barge-must-move-inadequate-permits/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://o.aolcdn.com/hss/storage/adam/a984bd7fcce54faf080c662b7a2c08a7/google-showroom-barge_thumbnail.jpg"/> Sure, Google's so-called mystery barge is a little less, well, mysterious now that we know it's meant to be "an interactive space where people can learn about new technology," but the tale of its creation continues the theme of intrigue. After ...]]></description>
      <dc:creator><![CDATA[Alexis Santos]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 20:56:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/google-mystery-barge-must-move-inadequate-permits/</guid>
    </item>
    <item>
      <title><![CDATA[Must See HDTV for the week of February 3rd: Walking Dead and Winter Olympics]]></title>
      <link>http://www.engadget.com/2014/02/03/must-see-hdtv-for-the-week-of-february-3rd-walking-dead-and-win/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://o.aolcdn.com/hss/storage/adam/8d512bb2704aa187cd0b969d927aa49/54736edc-5295-5b95-fdab-4f581fb052fa_TWD_408_GP_0730_0087_thumbnail.jpg"/> After the Super Bowl, we will have to fill the hours with non-football things (until the NFL Combine, at least) and right on time we have the return of AMC's The Walking Dead this weekend. If you prefer the rush of sports action (and the thrill of ...]]></description>
      <dc:creator><![CDATA[Richard Lawler]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 19:53:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/must-see-hdtv-for-the-week-of-february-3rd-walking-dead-and-win/</guid>
    </item>
    <item>
      <title><![CDATA[NudeAudio's Studio 5 wants to change how you think about Bluetooth speakers]]></title>
      <link>http://www.engadget.com/2014/02/03/nudeaudio-studio-5/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://o.aolcdn.com/hss/storage/adam/9a84c7c791b517f8b6b22d13f9cf6d49/nudestudio5wm_thumbnail.jpg"/> For many, the humble Bluetooth speaker has been marred by a preconception about poor audio quality (one that was often deserved until technologies like aptX gained traction). A problem usually compounded by generic, masculine design. NudeAudio wants ...]]></description>
      <dc:creator><![CDATA[James Trew]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 19:01:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/nudeaudio-studio-5/</guid>
    </item>
    <item>
      <title><![CDATA[Sprint's push-to-talk app adds support for six new phones]]></title>
      <link>http://www.engadget.com/2014/02/03/sprint-push-to-talk/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://o.aolcdn.com/hss/storage/adam/9ec4c7757e008b2ed549e34b6319d6e3/sprint_thumbnail.jpg"/> Sprint is bringing push-to-talk functionality to six new devices today: the Samsung Galaxy S4, Galaxy Note 3, LG G2, LG G Flex, LG Optimus F3, and the Kyocera Hydro Edge. The feature comes by way of Sprint's Direct Connect Now Android app, and allows ...]]></description>
      <dc:creator><![CDATA[Emily Price]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 18:37:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/sprint-push-to-talk/</guid>
    </item>
    <item>
      <title><![CDATA[SteamOS gets a native music player in Steam Music]]></title>
      <link>http://www.engadget.com/2014/02/03/steamos-steam-music/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://o.aolcdn.com/hss/storage/adam/f8db4547aac71e31579ec7c9955d6e02/steammusic_steamos_30pxhedimg_thumbnail.png"/> SteamOS still isn't really in wide use, and it's very firmly still in beta. Heck, the latest version still carries the disclaimer, "don't install it on any machine you are not prepared to lose." With Steam Music, though, SteamOS is one step closer to ...]]></description>
      <dc:creator><![CDATA[Ben Gilbert]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 18:04:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/steamos-steam-music/</guid>
    </item>
    <item>
      <title><![CDATA[White House will host its first Maker Faire later this year]]></title>
      <link>http://www.engadget.com/2014/02/03/white-house-maker-faire/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://o.aolcdn.com/hss/storage/adam/1e6a1bd0dd8156051fd2b1e92513a4bd/obama-and-joey-hudy_thumbnail.jpg"/> Many will tell you that Maker Faires inspire entrepreneurs, and the White House certainly agrees -- it wants to foster that inventive spirit across the US. Accordingly, it just unveiled plans to host the first-ever White House Maker Faire later in ...]]></description>
      <dc:creator><![CDATA[Jon Fingas]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 17:45:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/white-house-maker-faire/</guid>
    </item>
    <item>
      <title><![CDATA[Grand Theft Auto 5 is the best-selling game of 2013]]></title>
      <link>http://www.engadget.com/2014/02/03/gta-5-best-selling-2013/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://o.aolcdn.com/hss/storage/adam/986e733e54ec93cda8559d5808eb3dab/gta5_thatcash_thumbnail.jpg"/> Between a $1 billion debut and rave reviews, it's probably not a huge shock to you that Grand Theft Auto 5 was the best-selling game of 2013. Regardless, Rockstar Games' parent company Take-Two Interactive confirmed suspicions today in its financial ...]]></description>
      <dc:creator><![CDATA[Ben Gilbert]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 17:06:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/gta-5-best-selling-2013/</guid>
    </item>
    <item>
      <title><![CDATA[Paper news-reading app makes Facebook prettier, a bit less functional (hands-on)]]></title>
      <link>http://www.engadget.com/2014/02/03/facebook-paper-news-reading-app-hands-on/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://o.aolcdn.com/hss/storage/adam/6562459f3c92d437ad2a02a5be72713c/IMG_0042_thumbnail.jpg"/> Upon installing Facebook's newest app, you're greeted by a friendly female voice. "Welcome to Paper," she says. And, for many folks, Paper will be a welcome change indeed. But, before we tell you why Paper may be preferable to the full Facebook app, ...]]></description>
      <dc:creator><![CDATA[Michael Gorman]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 16:51:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/facebook-paper-news-reading-app-hands-on/</guid>
    </item>
    <item>
      <title><![CDATA[The NFL's new digital network is a step forward, but still not what cord-cutters want]]></title>
      <link>http://www.engadget.com/2014/02/03/nfl-now-cord-cutters/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://o.aolcdn.com/hss/storage/adam/17e74b884c240f8114a72829d0e2d136/nflnowlead2_thumbnail.jpg"/> Many people were excited by the announcement that this year's Super Bowl would be streamed online. But this came with a catch on smartphones -- the need to go through Verizon's NFL Mobile service rather than the Fox Sports Go app. And that's nothing ...]]></description>
      <dc:creator><![CDATA[Edgar Alvarez]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 16:35:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/nfl-now-cord-cutters/</guid>
    </item>
    <item>
      <title><![CDATA[Adobe's Photoshop guru John Nack is heading to Google's photography team]]></title>
      <link>http://www.engadget.com/2014/02/03/adobes-photoshop-john-nack-to-google/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://o.aolcdn.com/hss/storage/adam/29562d11c2642fb074f447e56477cfbd/photoshop_thumbnail.jpg"/> If you needed further proof that Google was serious about its photography efforts, one of Adobe's main Photoshop folks is heading for Mountain View to pitch in. John Nack, former Senior Product Manager for Photoshop, has been with the creative ...]]></description>
      <dc:creator><![CDATA[Billy Steele]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 15:44:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/adobes-photoshop-john-nack-to-google/</guid>
    </item>
    <item>
      <title><![CDATA[The government wants our cars to 'talk' to each other]]></title>
      <link>http://www.engadget.com/2014/02/03/the-government-wants-our-cars-to-talk-to-each-other/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://o.aolcdn.com/hss/storage/adam/bee8911d141b96aaa6415b73aecb9dd9/V2V-2_thumbnail.jpg"/> The US Department of Transportation (DOT) is throwing its weight behind vehicle-to-vehicle (V2V) communication systems. After years of experimentation and a real-world trial in Ann Arbor, MI the government is (almost) ready to make peer-to-peer ...]]></description>
      <dc:creator><![CDATA[Terrence O'Brien]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 15:11:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/the-government-wants-our-cars-to-talk-to-each-other/</guid>
    </item>
    <item>
      <title><![CDATA[Google Calendar now helps you remember holidays in 30 more countries]]></title>
      <link>http://www.engadget.com/2014/02/03/google-calendar-holidays-in-30-more-countries/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://o.aolcdn.com/hss/storage/adam/b6ffa5b05539272b8b22409ff7d82150/google-holiday-calendars_thumbnail.jpg"/> If you've ever made the faux pas of contacting foreign coworkers while they're out celebrating national holidays, you'll appreciate the latest update to Google Calendar. The service now accounts for special occasions in 30 more countries, including ...]]></description>
      <dc:creator><![CDATA[Jon Fingas]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 14:59:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/google-calendar-holidays-in-30-more-countries/</guid>
    </item>
    <item>
      <title><![CDATA[Facebook, Google and Yahoo now say when the US government requests user content (update: Microsoft too)]]></title>
      <link>http://www.engadget.com/2014/02/03/transparency-reports-now-cover-user-content-requests/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://o.aolcdn.com/hss/storage/adam/fa4c81883833fc832fda8809d121721c/google-fisa-content-requests_thumbnail.jpg"/> Last week, the US government finally relented on letting technology companies publish more detail about national security requests; today, some bigger firms are taking advantage of the looser rules. Facebook, Google, LinkedIn and Yahoo have updated ...]]></description>
      <dc:creator><![CDATA[Jon Fingas]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 14:23:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/transparency-reports-now-cover-user-content-requests/</guid>
    </item>
    <item>
      <title><![CDATA[Samsung's Galaxy Note Pro 12.2 ships in the US February 13th for $850, up for pre-order now]]></title>
      <link>http://www.engadget.com/2014/02/03/samsung-galaxy-note-pro-12-2-february-13th-for-850-/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://o.aolcdn.com/hss/storage/adam/a232f342ad50e55e533f2d954900c45d/IMG_1434_thumbnail.jpg"/> Arriving in the US just in time for Valentine's Day, the Samsung Galaxy Note Pro 12.2 is set to ship stateside February 13th with an equally beefy $850 price tag. If you'll recall, the 12.2-inch behemoth was announced back at CES and sports the same ...]]></description>
      <dc:creator><![CDATA[Billy Steele]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 13:55:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/samsung-galaxy-note-pro-12-2-february-13th-for-850-/</guid>
    </item>
    <item>
      <title><![CDATA[Mobile carrier Ting turns two, celebrates by lowering data prices]]></title>
      <link>http://www.engadget.com/2014/02/03/mobile-carrier-ting-turns-two-celebrates-by-lowering-data-price/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://o.aolcdn.com/hss/storage/adam/3eee3b767f7854186b5157ebf70bb7f2/htconepress_thumbnail.jpg"/> What would a birthday be without presents? Instead of receiving gifts, wireless provider Ting is celebrating its second birthday this week unwrapping lower data prices. Most data "buckets" on the service are now a bit cheaper, with the deepest ...]]></description>
      <dc:creator><![CDATA[Emily Price]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 13:45:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/mobile-carrier-ting-turns-two-celebrates-by-lowering-data-price/</guid>
    </item>
    <item>
      <title><![CDATA[Google finally opens Chromecast SDK, invites developers to get hackin']]></title>
      <link>http://www.engadget.com/2014/02/03/google-opens-chromecast-sdk/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://o.aolcdn.com/hss/storage/adam/94e4cb54166e3fc797b676b17e8b51bd/2-3-2014_chromecast_thumbnail.png"/> The Chromecast is already a pretty easy sell: It's $35 and brings most of the biggest names in streaming media to your TV with minimal hassle. So, now that Google has opened the SDK to developers, it's pretty much a no-brainer. While there are no ...]]></description>
      <dc:creator><![CDATA[Terrence O'Brien]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 13:30:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/google-opens-chromecast-sdk/</guid>
    </item>
    <item>
      <title><![CDATA[Google Now comes to the desktop in Chrome and Chrome OS betas]]></title>
      <link>http://www.engadget.com/2014/02/03/google-now-desktop/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://o.aolcdn.com/hss/storage/adam/cb1d3739ee6307e8f5b51a340e0cb410/2-3-2014_nowshot_thumbnail.png"/> Rumblings of Google Now on the desktop first started surfacing back in December of 2012, but it wasn't until a couple of weeks ago that it actually broke cover. If you were a user of the Canary build of the Chrome browser, you could enable the card ...]]></description>
      <dc:creator><![CDATA[Terrence O'Brien]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 12:50:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/google-now-desktop/</guid>
    </item>
    <item>
      <title><![CDATA[PlayStation 4 update supports Sony's pricey headsets, including a new model]]></title>
      <link>http://www.engadget.com/2014/02/03/playstation-4-update-supports-expensive-sony-headsets/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://o.aolcdn.com/hss/storage/adam/7447c5579c7addba797fdf8b4d265a40/ps4-gold-wireless-headset_thumbnail.jpg"/> PlayStation 3 owners who quickly moved to the PS4 couldn't take their expensive Pulse headsets with them, but that won't be a problem for long. A version 1.6 update reaching the newer console late this evening will bring full support for both the ...]]></description>
      <dc:creator><![CDATA[Jon Fingas]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 11:53:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/playstation-4-update-supports-expensive-sony-headsets/</guid>
    </item>
    <item>
      <title><![CDATA[Battlefield 4 gets even prettier with new AMD drivers]]></title>
      <link>http://www.engadget.com/2014/02/03/amd-mantle-driver-battlefield-4-benchmarks/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://o.aolcdn.com/hss/storage/adam/1d2892dfb18762f1989afb86ea5f2b19/bf4-construction-lead_thumbnail.jpg"/> If you caught our recent coverage of the huge Star Swarm demo, you'll know that AMD's Mantle programming tool has already proven itself capable of radically transforming a real-time strategy game. But the console-inspired API has been claimed to ...]]></description>
      <dc:creator><![CDATA[Sharif Sakr]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 11:22:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/amd-mantle-driver-battlefield-4-benchmarks/</guid>
    </item>
    <item>
      <title><![CDATA[Apple says Happy Birthday Mac with a video shot solely on iPhones]]></title>
      <link>http://www.engadget.com/2014/02/03/apple-says-happy-birthday-mac-with-iphone-video/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://o.aolcdn.com/hss/storage/adam/182c5d781a977021aae81716d853fe79/apple-124_thumbnail.jpg"/> Apple isn't done celebrating the Macintosh's 30th birthday just yet. To further mark the occasion, the folks in Cupertino dispatched 15 film crews to locales around the globe, armed with 100 iPhones to document just how far Apple has come in 30 ...]]></description>
      <dc:creator><![CDATA[Chris Velazco]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 10:58:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/apple-says-happy-birthday-mac-with-iphone-video/</guid>
    </item>
    <item>
      <title><![CDATA[Kentucky law could let kids swap foreign language classes for coding lessons]]></title>
      <link>http://www.engadget.com/2014/02/03/kentucky-code-language-credit/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://o.aolcdn.com/hss/storage/adam/f344dbc70789d2a3773ec3b6d66b48f/kentucky_senate_code_thumbnail.jpg"/> Concerned that not enough is being done to help kids with STEM (Science, Technology, Engineering and Math) subjects? Kentucky's Senate has just passed a bill that would count computer-programming classes as foreign language credits. That way, if ...]]></description>
      <dc:creator><![CDATA[Daniel Cooper]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 10:20:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/kentucky-code-language-credit/</guid>
    </item>
    <item>
      <title><![CDATA[UK retailer Ebuyer contemplates move from virtual to physical]]></title>
      <link>http://www.engadget.com/2014/02/03/ebuyer-uk-retail-hint/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://o.aolcdn.com/hss/storage/adam/2f6fd0fde92d749d69e8a11997507e62/ebuyer_thumbnail.jpg"/> After the collapses of Comet, Game, HMV and Blockbuster, the UK's high street has proven to be the elephant's graveyard of tech retailers. However, following Samsung's announcement that it'll open statement stores in the country, another business is ...]]></description>
      <dc:creator><![CDATA[Daniel Cooper]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 09:19:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/ebuyer-uk-retail-hint/</guid>
    </item>
    <item>
      <title><![CDATA[Microsoft squashes rumors of a cheaper, all-digital Xbox One]]></title>
      <link>http://www.engadget.com/2014/02/03/microsoft-squashes-rumors-of-a-cheaper-all-digital-xbox-one/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://www.blogcdn.com/www.engadget.com/media/2013/08/whitexboxone_thumbnail.jpg"/> Microsoft has no plans to build an Xbox One sans Blu-ray disc for $100 less than the current model, according to a tweet by Xbox chief Aaron Greenberg. Rumors about a cheaper, all-digital version with a 1TB drive floated up recently from the NeoGAF ...]]></description>
      <dc:creator><![CDATA[Steve Dent]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 08:13:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/microsoft-squashes-rumors-of-a-cheaper-all-digital-xbox-one/</guid>
    </item>
    <item>
      <title><![CDATA[Sound system simulates a rocket blast, would kill you just as dead]]></title>
      <link>http://www.engadget.com/2014/02/03/esa-leaf-accoustic-rocket-test-facility/?ncid=rss_truncated</link>
      <description><![CDATA[<img src="http://o.aolcdn.com/hss/storage/adam/cca5bc038119e00f92dd95d2296ec647/large-european-acoustic-facility-speaker-2014-02-03-01_thumbnail.jpg"/> Sure, your stereo may go up to 11 and beyond, but you probably couldn't murder your listeners with it. The European Space Agency has a bigger budget, however, and its Large European Acoustic Facility (LEAF) has a 36-foot wide by 54-foot high wall of ...]]></description>
      <dc:creator><![CDATA[Steve Dent]]></dc:creator>
      <pubDate>Mon, 03 Feb 2014 06:20:00 -0500</pubDate>
      <guid isPermaLink="false">http://www.engadget.com/2014/02/03/esa-leaf-accoustic-rocket-test-facility/</guid>
    </item>
  </channel>
</rss>