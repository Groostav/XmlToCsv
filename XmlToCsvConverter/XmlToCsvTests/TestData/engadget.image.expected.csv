url,title,link,channel_Id
"http://www.blogsmithmedia.com/www.engadget.com/media/feedlogo.gif","Engadget","http://www.engadget.com",0
